src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html

```diff
@@ -365,7 +365,33 @@ <h4 *ngIf="isTestRun">
                             "
                         >
                             <div class="alert alert-info">
-                                {{ 'artemisApp.exerciseAssessmentDashboard.noSubmissionsBecauseOfOtherLockedSubmissions' | translate }}
+                                <span
+                                    jhiTranslate="artemisApp.exerciseAssessmentDashboard.noSubmissionsInfo"
+                                    [translateValues]="{
+                                        freeAssessments:
```

Comment:
please use a get function in the typescript file to calculate the `freeAssessments`. Otherwise this is a little bit hard to read.
Also for the other calculated values

Reply:
we try to avoid function calls, as due to angular change tracking it would be evaluated many many times while loading the component, so I guess this is the best way to handle it